src/utils/database_util.py

```diff
@@ -50,15 +52,41 @@ def table_leetcodequestion_delete(self, question_id: int):
         table = self.db.table(self.TABLE_LEETCODE_QUESTION)
         return table.remove(where("id") == question_id)
 
-    def table_leetcodeuser_insert(self, items: List[dict]) -> int:
+    def table_leetcodeuser_insert(self, item: dict) -> bool:
         """
         Inserts a collection of items in the LeetcodeUser database table
         """
         table = self.db.table(self.TABLE_LEETCODE_USER)
-        inserts = 0
-        for item in items:
-            matches = table.search(where("id") == item["id"])
-            if len(matches) == 0:
-                table.insert(item)
-                inserts += 1
-        return inserts
+        matches = table.search(where("discord_id") == item["discord_id"] \
+        or where("leetcode_id") == item["leetcode_id"])
```

Comment:
Please indent the `or where(...)` line by 4 spaces. It is common for multiline code to indent lines after the first one.